docs/language/contracts.rst

```diff
@@ -125,6 +138,25 @@ The sequence of the accounts in the ``AccountMeta`` array matters and must follo
 :ref:`IDL ordering <account_management>`.
 
 
+.. _solana_contract_call:
+
+Calling a contract on Solana
+____________________________
+
+On Solana, contracts are deployed to a program account, which holds only the contract's executable binary. For this
+reason, a Solidity contract on Solana is represented by its program account address. When making an external call,
+the contract definition serves only as a interface to provide the functions' signatures. As the program account address,
```

Comment:
"When making an external call, the contract definition serves only as a interface to provide the functions' signatures"

I have no idea what this means.